tests/related_resource/api/resources.py

```diff
@@ -278,3 +280,24 @@ class Meta:
         queryset = Contact.objects.prefetch_related('groups')
         resource_name = 'contact'
         authorization = Authorization()
+
+
+class ConstantResource(ModelResource):
+    class Meta:
+        always_return_data = True
+        queryset = Constant.objects.all()
+        authenticaion = Authentication()
+        authorization = ReadOnlyAuthorization()
+
+
+class EquationResource(ModelResource):
+    description = fields.CharField()
+    constants = fields.ToManyField(
+        ConstantResource, 'constants', related_name='equations', null=True,
+        blank=True, full=True)
```

Comment:
That won't be possible since the related resource has ReadOnlyAuthorization. If you want ConstantResource to have ReadOnlyAuthorization, but for EquationResource.constancts to be changeable, you'll need to create another resource with a different Authorization that permits creating/updating.

    class ConstantResource(ModelResource):
        class Meta:
            always_return_data = True
            queryset = Constant.objects.all()
            authenticaion = Authentication()
            authorization = ReadOnlyAuthorization()


    class WritableConstantResource(ConstantResource):
        class Meta(ConstantResource.Meta):
            authorization = AMorePermissibleAuthorization()


    class EquationResource(ModelResource):
        description = fields.CharField()
        constants = fields.ToManyField(
            WritableConstantResource, 'constants', related_name='equations', null=True,
            blank=True, full=True)